packages/scene-composer/src/components/three-fiber/AnimationComponent/AnimationComponent.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment } from 'react';
+import * as THREE from 'three';
+import {  useFrame } from '@react-three/fiber';
+import { GLTF } from 'three/examples/jsm/loaders/GLTFLoader';
+import { useGLTF } from '../ModelRefComponent/GLTFLoader';
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+import { ISceneNodeInternal, useEditorState, useViewOptionState } from '../../../store';
+
+import { IAnimationComponent } from '../../../interfaces';
+
+function pauseAnimation(gltfModel, AnimationMixer, isPaused, AnimationList){
+  try{
+    AnimationList.forEach( function ( clip ) {
+      const animation = THREE.AnimationClip.findByName( gltfModel, clip );
+      AnimationMixer.clipAction(animation).paused = isPaused
+    } );    
+  }
+  catch (error) {
+   // console.error(error);
+  }
+}
+
+function Animate(gltfModel, AnimationMixer, AnimationList){
+  try{
+    AnimationList.forEach( function ( clip ) {
+      const animation = THREE.AnimationClip.findByName( gltfModel, clip );
+      AnimationMixer.clipAction(animation).play();
+    } );
+  }
+  catch (error) {
+    //console.log(error)
+  }
+    useFrame((state, delta) => { 
+    AnimationMixer?.update(delta)
+  });
+}
+export const isPaused = false;
+
+export function getClip(uri){
```

Comment:
Is this repulling this from s3 via a URI? If the gltf has already be downloaded by the model system we don't want to re-download it. We'd rather have use a reference to the existing model.